.github/workflows/test.yml

```diff
@@ -23,3 +23,6 @@ jobs:
         env:
           NODE_OPTIONS: '--experimental-modules --es-module-specifier-resolution=node'
         run: yarn test
+
```

Comment:
Let's improve github integration with extracting `e2e` to separate workflow file and enabling tmp

storage of recordings https://github.com/cypress-io/github-action#artifacts 

Video Talk:
-  https://www.youtube.com/watch?v=gokM_zEmWLA